conf/messages

```diff
@@ -34,15 +34,6 @@ error.ducr.format = DUCR number is in incorrect format
 error.mucr.empty = MUCR number cannot be empty
 error.mucr.format = MUCR number is in incorrect format
 
-index.title =Export Declarations
```

Comment:
These are indeed un-used

Reply:
I could not find any page that still uses message keys like index.*.

Reply:
Indeed, you are correct.  I double checked to save anyone else doing so.